test/docker-e2e/e2e_test.go

```diff
@@ -22,7 +23,22 @@ import (
 const (
 	txsimImage = "ghcr.io/celestiaorg/txsim"
 	txSimTag   = "v4.0.10-arabica"
-	homeDir    = "/var/cosmos-chain/celestia"
+
+	// Liveness check constants
+	defaultBlocksPerValidator = 3                    // Minimum blocks each validator should propose for liveness validation
+	validatorAddrTruncateLen  = 8                    // Length to truncate validator addresses for display
```

Comment:
[question] Why truncate the address to 8 characters? Seems possible to print the entire address.